app/src/main/java/com/codesoom/assignment/application/TaskService.java

```diff
@@ -51,4 +51,9 @@ private Long generateId() {
         newId += 1;
         return newId;
     }
+
+
+   public void deleteAllTasks(){
```

Comment:
```suggestion
   public void deleteAllTasks() {
```
여기도 이렇게 빈 칸 하나가 들어갔네요